AIMSICD/src/main/AndroidManifest.xml

```diff
@@ -26,6 +26,11 @@
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
 
     <!-- SPECIAL PERMISSIONS TO BE ADDED AFTER THIS LINE. /> -->
+    
+    <!-- Required: Used to deliver privacy-aware Bugsnag crash reports -->
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <!-- Optional to see which Activity was active at the time of a crash -->
+    <uses-permission android:name="android.permission.GET_TASKS"/>
```

Comment:
When my fork is ready we don't need to declare the permissions here (they will be merged from the bugsnag library during the build, see bugsnag/bugsnag-android#90)